### ISSUE TITLE
FontAwesome escaped inside <Text />

### ISSUE THREAD

**Issue by robbederks** [lavrton/react-konva#108]
Hi,

First of all, thank you for the great library,

If I make a Text-component in which I want to use a FontAwesome icon in this manner:
```
const text = <Text
            x={10}
            y={10}
            fontFamily="FontAwesome"
            fill="#000"
            text="\uf002"
        />;
```
the backslash in the text gets escaped automatically (to `\\uf002`). As a similar use case is displayed in standard Konva, I suppose this should also work in React-Konva.

Best regards

**Comment by lavrton** [comment]
Pass text as javascript string and it will work just fine:
```js
class App extends React.Component {
    componentDidMount() {
      // force redraw after a timeout
      // to make sure that the font is loaded
      // setTimeout is just for demo
      // it is better to use some font observer
      setTimeout(() => {
        this.layer.draw();
      }, 2000);
    }
    render() {
    return (
      <Stage width={700} height={700}>
        <Layer ref={node => this.layer = node}>
            <Text
            x={10}
            y={50}
            fontFamily="FontAwesome"
            text={'\uf063'}
            fill="black"
        />
        </Layer>
      </Stage>
    );
    }
}
```

Demo: http://jsbin.com/tipedewebi/1/edit?html,js,output

Also, make sure your font is loaded when you want to draw a text. (just redraw stage when font is loaded).

**Comment by michaelwilson** [comment]
So this is very awesome, but I've been having a couple of problems:
. I'm having problems getting the "pro" Duotone font styles.
. Using setFontStyle to change the font weight, which should change the font style, but it's not working.

[Here](https://fontawesome.com/how-to-use/on-the-web/advanced/css-pseudo-elements) you can see that the font styles "Solid", "Regular", "Like" are used to select different icon styles. 

Here's the code I use to create the icon: ' '\uf005' is FontAwesome's "star", and '900' (or 900) should get me the Duotone star. 

`      
let overlay_icon_star = new Konva.Text({id: 'overlay_icon_star_' + photograph_index.toString(),
                                               x: overlay_icon_star_x,
                                               y: overlay_icon_y,
                                               text: '\uf005',
                                               fontSize: 16,
                                               fontFamily: '"FontAwesome"',
                                               fontStyle: '900',
                                               fill: '#3097d1',
                                               opacity: 0,
                                               align: 'center',
                                               verticalAlign: 'middle',
                                               height: overlay_icon_group_height,
                                               width: overlay_icon_width
                                               });
`
I don't get a duotone start, I just get an outline star.

In an "onClick" method, I'll do this:
`
...
  event.target.setFontStyle('400');
 stage.draw();
...
`
To get a solid or outline star, but it never changes.
 
On the same page as the canvas, I refer to other duotone icons using <i ... > and they work perfectly.

So, to sum up:
. I can't figure out how to access "pro" icons from Konva.
. I can't seem to influence which icon style is used by using setIconStyle to set the icon weight.

How can I figure this out?

**Comment by lavrton** [comment]
Can you make a demo of your issue?

**Comment by michaelwilson** [comment]
Yes, let me do some more debugging and upgrade to V7.
